### DISCUSSION TITLE
How to set text-to-speech

### DISCUSSION THREAD

**Discussion by foxTr0t1178** [lencx/ChatGPT#959] [General]
I can't seem to figure out the text-to-speech function. I see the little audio icon appear on the generated text, it appears to be a button, but it doesn't do anything when I click on it. Is there any way to fix this in the config? Or am I doing something wrong?

**Comment by furopi** [comment]
Also confused here... The app is great though, thanks alot! 😍

**Comment by gmy77** [comment]
I wait for this... like in the web, the MIC button...
